### DISCUSSION TITLE
Do you still support zones?  Is this feature gonna be continued? 

### DISCUSSION THREAD

**Discussion by brunoreis** [vercel/next.js#10784] [Help]
I was willing to try the usage of zones. But I realised they are not present on the most recent docs. Please, are they gonna be discontinued?

**Comment by timneutkens** [comment]
> are they gonna be discontinued

They're not going to be discontinued as they're sort of like a "meta feature", it's not a feature you enable / disable. The only thing related to it is [`assetPrefix`](https://nextjs.org/docs/api-reference/next.config.js/cdn-support-with-asset-prefix) as you'd want to have 2 separate paths for the assetPrefix of 2 apps. Linking between them already works as Next.js will force a server-side transition when a page doesn't exist within the app.

An updated doc just hasn't been written yet and will be added sometime soon.
